Mod2/Lessons/Week1/IntroToDatabases.md

```diff
@@ -0,0 +1,138 @@
+
+We link the two tables by using a **foreign key** on one or more tables.  In this case, we use `Customer Id` as a foreign key on the Orders, that refers back to the primary keys from Customers.  
+
+> With a partner, practice describing these two tables.  Use as many vocab words as you can: data, table, primary key and foreign key.
+
+## Create Database
+<!-- - Download Postgresql
+- Open PgAdmin
+- Create Database with one table -->
+
+So far, we have been working with representations of what data might look like inside of a database.  We have seen data organized into tables, using primary and foreign keys to relate to records in other tables.  Now, we are going to create a database.  This database will live on your computer (it will be accessible only by you, or someone with your laptop).
+
+### Download PostgreSQL
+
+There are [many different databases](https://appinventiv.com/blog/top-web-app-database-list/) that are available.  During your time at Turing, we will primarily work with [PostgreSQL](https://www.postgresql.org/).
+
+1. Download the [Windows x86-64 PostgreSQL Installer](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+    * Choose the installer for the most current version (the highest number).
+    * It will take a few minutes for the installer to download.
+2. Open the installer file to start the setup Wizard.  
+            ![](/Mod2/Images/Week1/postgresqlInstallerWizard.png)
+3. Click 'Next' through the wizard, keeping all default settings. You will need to enter and confirm a password on the 4th or 5th step - this is the only customization you will do.
+4. On the last step, before you click 'Finish', uncheck the box so that Stack Builder is **not** launched upon completion.
+            ![](/Mod2/Images/Week1/postgresqlInstallerWizardLastStep.png)
+5. Open PgAdmin in preparation for the next section
+            ![](/Mod2/Images/Week1/OpenPGAdmin.png)
+
+**If you have any issues during this installation, reach out to an instructor!**
+
+### Create a Database with PgAdmin
+
+** Instructor Note ** You could walk the students through the database and table creation, or let them explore on their own.  We do not need them to be _experts_ in database administration.
+
+Once in PgAdmin, we are going to create a database for our customer and order information.
+
+> In small groups, work through the following steps.  You should each complete the steps individually, but use your group-mates as resources if you get stuck.  And, if you need an instructor's help - reach out!  
+> 1. Right Click on `Databases` and `Create` a `Database`
+> ![](/Mod2/Images/Week1/CreateDb1.png)
+> 2. Give your database a name and `Save`
+> ![](/Mod2/Images/Week1/CreateDb2.png)
+> 3. If successful, your pgAdmin should look like: 
+> ![](/Mod2/Images/Week1/CreateDb3.png)
+
+Now that we have a database created, we need to add two tables to our database for `customers` and `orders`
```

Comment:
Does this require a separate installation, or has pgAdmin been installed in a previous step?